src/styles/_themes.scss

```diff
@@ -175,35 +181,35 @@ $themes: (
 
     colorBtnText: #212121,
     colorBtnBackground: transparent,
-    colorBtnSvgFill: $colorSmoke,
+    colorBtnSvgFill: #666,
 
-    colorBtnTextHover: $colorWhite,
-    colorBtnBackgroundHover: #dadada,
-    colorBtnSvgFillHover: $colorWhite,
+    colorBtnTextHover: #212121,
```

Comment:
Why not define these colors (#212121, #666) in src/styles/_variables.scss and use the variables here ?

Reply:
That is the plan. We just aren't doing it yet. As part of the broader central alignment we will be sharing sass variables files which can be used here though.